components/eamxx/docs/user/clean_clear_sky.md

```diff
@@ -0,0 +1,83 @@
+# Clean- and clean-clear-sky diagnostics
+
+In order to decompose the aerosol effective radiative forcing, additional diagnostic radiation calls are needed.
+These extra diagnostics are optionally added to the main radiation call. The extra diagnostics are:
+
+- Clean-clear-sky fluxes: the fluxes that would be present if there were neither aerosols nor clouds, and are calculated by adding an additional radiation call at the very beginning of the logic before the optics class is endowed with aerosol and cloud properties.
+- Clean-sky fluxes: the fluxes that would be present if there were no aerosols, and are calculated by adding an additional radiation call after substantiating an additional optics class, but not endowing it with aerosol properties.
+
+It was necessary to add an additional optics class because the original optics class is endowed with aerosols before clouds (in order to calculate the clear-sky fluxes).
+
+## Example setup (current as of April 2024)
+
+The extra calls are controlled by runtime flags `extra_clnclrsky_diag` and `extra_clnsky_diag` (they take either `true` or `false` as their values).
+
+```shell
+    ./atmchange extra_clnclrsky_diag=true
+    ./atmchange extra_clnsky_diag=true
+```
+
+An example output file with the additional radiation diagnostics requested.
+
```

Comment:
I think it would make more sense to include an example that *only* includes clnclrsky variables. As is, the important bits are hidden in a pile of irrelevant stuff and the example will take more screen real estate than it needs to. But again, I'm fine with keeping it as is if you want.